Megaparsec/Megaparsec.lean

```diff
@@ -1,7 +1,6 @@
 namespace Megaparsec
 
-structure Pos where
-  pos : Nat
+abbrev Pos := Nat
```

Comment:
Will it let me use `Pos` and `Nat` interchangably? If so, it's not quite what we want. We want to distinguish parse positions from naturals.